### DISCUSSION TITLE
Tero Subtitler 1.0.1.5

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#228] [Announcements]
We've been hard at work making **Tero Subtitler** better for you. This includes bug fixes and performance enhancements. 
For more detailed information, check the [change log](https://raw.githubusercontent.com/URUWorks/TeroSubtitler/main/changelog.txt).

Note: If the Windows installer doesn’t work, it may be necessary to temporarily disable antivirus protection.

**Use it at your own risk**!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.1.5'>Tero Subtitler 1.0.1.5</a>.</em>

**Comment by stoleto** [comment]
Очень нравится программа, спасибо! Только не понял, где активировать аудиоволну. И ещё хотелось бы видео без черных полос внизу и вверху, чтобы сразу видеть, где новые субтитры будут конфликтовать с вшитыми титрами. Ну и кнопку "Переместить запись" в окне "Видео" не нашёл.
![1](https://github.com/URUWorks/TeroSubtitler/assets/157589230/0ca2c162-d91e-4c6e-9335-743e9d7660bf)

**Reply by chenlung** [reply]
Please use Discussions for questions.

Regarding waveform, you need to generate it (in the Video menu).

**Comment by Natcarol2000** [comment]
Hello, I've download the Subtitler but when trying to extract the waveform I get and error message saying: **unable to find "".**

Does anyone have an idea of what is going on? How can I fix this issue and extract the waveform? (sorry, I did not find where to enter the question in the Discussions tab).

**Reply by URUWorks** [reply]
This is because you do not have the FFMPEG path configured in settings.

**Reply by Natcarol2000** [reply]
ok, thanks. I'll configure this path in settings
